bin/dartdoc.dart

```diff
@@ -20,24 +20,30 @@ void main(List<String> arguments) {
   }
   List<String> excludeLibraries =
       results['exclude'] == null ? [] : results['exclude'].split(',');
+  String sdkLocation = results['dart-sdk'];
+  if (sdkLocation == null) {
+    _printUsageAndExit(parser);
+  }
 
   String url = results['url'];
   var currentDir = Directory.current;
-  new DartDoc(currentDir, excludeLibraries, url).generateDocs();
+  new DartDoc(currentDir, sdkLocation, excludeLibraries, url).generateDocs();
 }
 
 /// Print help if we are passed the help option or invalid arguments.
 void _printUsageAndExit(ArgParser parser) {
   print(parser.usage);
-  print('Usage: dartdoc [OPTIONS]');
+  print('Usage: dartdoc --dartsdk=sdkLocation [OPTIONS]');
```

Comment:
Agreed, looking into it. Would prefer not to have the sdk location as a requirement. Added this so that dartdoc can be used for now.